src/tools/database/export-demo.ts

```diff
@@ -34,18 +35,18 @@ const contactAnonymisers = [
   contactProfileAnonymiser,
   paymentsAnonymiser,
   paymentDataAnonymiser
-] as ModelAnonymiser<unknown>[];
+] as ModelAnonymiser[];
```

Comment:
why not like this?

```ts
const calloutsAnonymisers: ModelAnonymiser[] = [
  calloutsAnonymiser,
  calloutTagsAnonymiser
];
```

Same for the other anonymisers.

Reply:
I'm not sure why but it gives a type error:

```
Type 'ModelAnonymiser<ContactRole>' is not assignable to type 'ModelAnonymiser<ObjectLiteral>'.
  Types of property 'objectMap' are incompatible.
    Type 'ObjectMap<ContactRole>' is not assignable to type 'ObjectMap<ObjectLiteral>'.
      Property 'contactId' is incompatible with index signature.
        Type 'PropertyMap<string>' is not assignable to type 'PropertyMap<any> | undefined'.ts(2322)
```

I could change it to:
```ts
const calloutAnonymisers: ModelAnonymiser<any>[] = [...]
```

In general I need to rework the anonymisation code so am tempted to leave this with the casting for now

Reply:
okay then I would leave it like this for the time being